src/modules/asab-config/ConfigContainers/ConfigContainer.js

```diff
@@ -13,60 +13,204 @@ import { useTranslation } from 'react-i18next';
 import { TreeViewComponent } from "./TreeViewComponent";
 import ConfigEditor from "./ConfigEditor";
 import ConfigList from "./ConfigList";
-// import reducer from './reducer';
+import ConfigImport from "./ConfigImport";
 
 function ConfigContainer(props) {
 
-	const { t, i18n } = useTranslation();
+	const App = props.app;
+	const ASABConfigAPI = App.axiosCreate('asab_config');
```

Comment:
@Aringoaway Its better to use `props.app.axiosCreate(...)` that we know the full path from where it is taken